xclim/indices/_threshold.py

```diff
@@ -2111,6 +2112,47 @@ def tropical_nights(
     return tn_days_above(tasmin, thresh=thresh, freq=freq)
 
 
+@declare_units(pr="[precipitation]", prc="[precipitation]", thresh="[precipitation]")
+def rprctot(
+    pr: xarray.DataArray,
+    prc: xarray.DataArray,
+    thresh: str = "1.0 mm/day",
+    freq: str = "YS",
+) -> xarray.DataArray:
+    """Proportion of accumulated precipitation arising from convective processes.
+
+    Return the proportion of total accumulated precipitation due to convection on days with total precipitation exceeding a specified threshold during the given period.
+
+    Parameters
+    ----------
+    pr : xarray.DataArray
+      Daily precipitation.
+    prc : xarray.DataArray
+      Daily convective precipitation.
+    thresh : str
+      Precipitation value over which a day is considered wet.
+    freq : str
+      Resampling frequency.
+
+    Returns
+    -------
+    xarray.DataArray, [time]
+      The proportion of the total precipition accounted for by convective precipitation for each period [1].
+    """
+
+    thresh = convert_units_to(thresh, pr, "hydro")
+    prc = convert_units_to(prc, pr)
+
+    wd = compare(pr, ">=", thresh)
+    pr_tot = pr.where(wd).resample(time=freq).sum(dim="time")
+    prc_tot = prc.where(wd).resample(time=freq).sum(dim="time")
```

Comment:
```suggestion
    pr_tot = rate2amount(pr).where(wd).resample(time=freq).sum(dim="time")
    prc_tot = rate2amount(prc).where(wd).resample(time=freq).sum(dim="time")
```
You can't sum a flux! Another equivalent solution would be use the mean on the period, rather than the sum.